### ISSUE TITLE
Tests for background-* computed value doesn't seem to make sense.

### ISSUE THREAD

**Issue by emilio** [web-platform-tests/wpt#17925]
They were added by Chromium in https://github.com/web-platform-tests/wpt/pull/17015.

The spec quotes mention how the values are used, sure.

But I think the Firefox behavior is the right one. The Blink and WebKit conversions are lossy.

The computed value has to have all the specified values, even if they're for empty layers. This is also the case in Blink and WebKit. They just don't serialize them via getComputedStyle().

@ericwilligers @lilles mind elaborating on what parts of the spec do those tests follow? Why does serialization of background-position e.g. has to depend on the computed value of background-image? That doesn't make sense to me.

**Comment by ewilligers** [comment]
I was planning to create a CSS WG issue for this question.

For the test [background-blend-mode-computed.html](https://wpt.fyi/results/css/compositing/parsing/background-blend-mode-computed.html?label=master&label=experimental), I followed the Firefox result where the number of layers in the getComputedStyle result matches the number of layers in the specified value.

For the [tests](https://wpt.fyi/results/css/css-backgrounds/parsing/background-computed.html?label=master&label=experimental) of the properties in the Background and and Borders spec, I followed Blink and WebKit where the number of layers in the getComputedStyle result matches the number of layers in background-image.

**Comment by emilio** [comment]
Yeah, let's file a spec issue, I guess it's better to have broader discussion ablut this.

Animations and transitions behave consistently across browsers iirc, matching FF, but I could be misremembering (I'm OOO today)
